### PR TITLE
Project permissions and changes to create project flow

### PR DESCRIPTION
**Pull request by Preeticp**
This PR:

- Adds a module for providing permissions to Projects
- Modifies the create project flow

This applies to the 4.3 version
This has been reviewed by SME and QE

### CODE REVIEW

**Comment by Preeticp** [comment]
@ahardin-rh request you to review this PR. Thank you.

File: modules/odc-creating-projects-using-developer-perspective.adoc [hunk 1]
```diff
@@ -21,5 +21,5 @@ image::odc_create_project.png[Create Project]
 . In the *Create Project* dialog box, enter a unique name for the *Name* field. For example, enter `myproject` as the name of the project in the *Name* field.
 . Optionally, add the *Display Name* and *Description* details for the Project.
 . Click *Create*.
-
-Your project is now created and you can add applications and other workloads in to this project.
+. Navigate to the *Advanced → Project Details* page to see the dashboard for your project.
+. In the *Project* drop-down menu at the top of the screen, select *all projects* to list all the projects in your cluster. If you have adequate permissions for a project you can use the  *Options* menu {kebab} to edit or delete the project.
```
Comment (ahardin-rh) [inline modules/odc-creating-projects-using-developer-perspective.adoc]: all of the projects

Comment (ahardin-rh) [reply modules/odc-creating-projects-using-developer-perspective.adoc]: project, you can

File: modules/odc-providing-project-permissions-using-developer-perspective.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+You can also use:
+
+* The *Select a role* drop-down menu, to modify the access permissions of an existing user.
+* The *Remove Access* icon, to completely remove the access permissions of an existing user to the project.
+
+[NOTE]
+====
+Advanced role based access control is managed in the *Roles* and *Roles Binding* views in the *Administrator* perspective.
```
Comment (ahardin-rh) [inline modules/odc-providing-project-permissions-using-developer-perspective.adoc]: role-based access

**Comment by Preeticp** [comment]
@ahardin-rh I have fixed all the comments and squashed the commits, can you merge this, please?

**Pull request merged**